update.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+set -e -u
+
+MB_RNGPY_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+cd "$MB_RNGPY_ROOT"
+
+usage="Usage: $0 [<python command>]"
+
+if [ $# -gt 1 ]
+then
+  echo Error: Too many arguments
+  echo $usage
+  exit -1
+fi
+
+if [ $# -eq 1 ]
+then
+  python_command=$1
+  if ! type $python_command &>/dev/null
+  then
+    echo Error: $python_command command not found
+    echo $usage
+    exit -1
```

Comment:
Nitpick: the only valid exit statuses are 0-255 (https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html), with >125 are reserved by the shell. -1 will be treated as 255. Just 1 should be enough to indicate failure here.

Reply:
Replaced with 64 (`EX_USAGE`) and 69 (`EX_UNAVAILABLE`) for neatness!